pkgs/tools/security/sbomnix/default.nix

```diff
@@ -1,53 +1,209 @@
+
+    wrapProgram $out/bin/nix_outdated \
+        --prefix PATH : ${lib.makeBinPath [nix-visualize]}
+
+    wrapProgram $out/bin/provenance \
+        --prefix PATH : ${lib.makeBinPath [nix]}
+
+  '';
+
+  # We need to patch vulnix to add support for runtime-only scan
+  # ('-C' command-line option) which is currently not available in
+  # released version of vulnix.
+  # Pending: https://github.com/nix-community/vulnix/pull/80.
+  vulnix-patched = vulnix.overrideAttrs (
+    old: rec {
+      patches = (old.patches or [ ]) ++ [
+        ./patches/vulnix/0001-remove-unused-function-parameter.patch
+        ./patches/vulnix/0002-add-flag-to-scan-only-runtime-dependencies-store-out.patch
+        ./patches/vulnix/0003-rename-runtime-to-closure.patch
+        ./patches/vulnix/0004-add-shorthand-C-for-closure.patch
+        ./patches/vulnix/0005-improve-comment.patch
+        ./patches/vulnix/0006-fix-failure-to-find-deriver-for-derivation-input-sou.patch
+        ./patches/vulnix/0007-avoid-unnecessary-nix-invocation-during-closure-scan.patch
+      ];
+    }
+  );
+
+  # https://github.com/craigmbooth/nix-visualize (not available in nixpkgs)
+  nix-visualize = python.pkgs.buildPythonApplication {
+    version = "1.0.5";
+    pname = "nix-visualize";
+    format = "setuptools";
+
+    src = fetchFromGitHub {
+      owner = "craigmbooth";
+      repo = "nix-visualize";
+      rev = "5b9beae330ac940df56433d347494505e2038904";
+      hash = "sha256-VgEsR/Odddc7v6oq2tNcVwCYm08PhiqhZJueuEYCR0o=";
+    };
+
+    propagatedBuildInputs = [
+      matplotlib
+      networkx
+      pygraphviz
+      pandas
+    ];
+  };
+
+  # https://github.com/thombashi/df-diskcache (not available in nixpkgs)
+  dfdiskcache = python.pkgs.buildPythonPackage rec {
+    version = "0.0.2";
+    pname = "df-diskcache";
+    format = "setuptools";
+
+    src = fetchFromGitHub {
+      owner = "thombashi";
+      repo = "df-diskcache";
+      rev = "v${version}";
+      hash = "sha256-s+sqEPXw6tbEz9mnG+qeUSF6BmDssYhaDYOmraFaRbw=";
+    };
+
+    propagatedBuildInputs = [
+      pandas
+      simplesqlite
+      typing-extensions
+    ];
+
+    pythonImportsCheck = [ "dfdiskcache" ];
+    doCheck = false;
+  };
+
+  # Required due to dfdiskcache (not available in nixpkgs)
+  simplesqlite = python.pkgs.buildPythonPackage rec {
+    version = "1.5.2";
+    pname = "SimpleSQLite";
+    format = "setuptools";
+
+    src = fetchFromGitHub {
+      owner = "thombashi";
+      repo = "SimpleSQLite";
+      rev = "v${version}";
+      hash = "sha256-Yr17T0/EwVaOjG+mzdxopivj0fuvQdZdX1bFj8vq0MM=";
+    };
+
+    propagatedBuildInputs = [
+      dataproperty
+      mbstrdecoder
+      pathvalidate
+      sqliteschema
+      tabledata
+      typepy
+    ];
+
+    pythonImportsCheck = [ "simplesqlite" ];
+    doCheck = false;
+  };
+
+  # Required due to dfdiskcache (not available in nixpkgs)
+  sqliteschema = python.pkgs.buildPythonPackage rec {
```

Comment:
This is nixpkgs, so it needs to be added to the python package set (and needs `meta`).